### PR TITLE
Multi year Osse eligibility marking UI

### PR DESCRIPTION
**Pull request by harshaellanki**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [x] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update version)

# What is the ticket # detailing the issue?

Ticket: 
https://www.pivotaltracker.com/story/show/185424878
https://www.pivotaltracker.com/story/show/185424914

# A brief description of the changes

Current behavior:

New behavior:

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable that is used to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [x] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

### CODE REVIEW

File: app/models/benefit_coverage_period.rb [hunk 1]
```diff
@@ -283,6 +283,9 @@ def all
       organizations.size > 0 ? organizations.first.hbx_profile.benefit_sponsorship.benefit_coverage_periods : nil
     end
 
+    def osse_eligibility_years_for_display
```
Comment (nks2109) [inline app/models/benefit_coverage_period.rb:286]: Is it possible to get the values from settings/RR ?

File: app/models/services/ivl_osse_eligibility_service.rb [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Services
+  # service for ivl osse actions
+  class IvlOsseEligibilityService
+
+    attr_reader :person, :role, :args
+
+    def initialize(params)
+      @person = Person.find(params[:person_id])
```
Comment (nks2109) [inline app/models/services/ivl_osse_eligibility_service.rb:10]: can you validate the person_id before finding the person with it?

File: components/benefit_markets/app/models/benefit_markets/benefit_market_catalog.rb [hunk 3]
```diff
@@ -137,6 +137,10 @@ def open_enrollment_minimum_begin_day_of_month(use_grace_period = false)
       open_enrollment_end_on_day - minimum_length
     end
 
+    def self.osse_eligibility_years_for_display
+      [2022, 2023, 2024]
```
Comment (nks2109) [inline components/benefit_markets/app/models/benefit_markets/benefit_market_catalog.rb:141]: similarly here as well

**Review by raghuramg — APPROVED** [review]
code changes looks good to me

**Review by nks2109 — APPROVED** [review]
addressed previous comments and code looks good

**Pull request closed without merging**